holoviews/plotting/bokeh/element.py

```diff
@@ -154,6 +154,12 @@ class ElementPlot(BokehPlot, GenericElementPlot):
        elements and the overlay container, allowing customization on a
        per-axis basis.""")
 
+    subcoordinate_y = param.ClassSelector(default=False, class_=(bool, tuple), doc="""
+       Enables sub-coordinate scales for this plot.""")
```

Comment:
Maybe add a small sentence about what a tuple means.

Reply:
I added a comment.

Why are the docstring bodies indented with 3 blank spaces instead of 4? :D

Reply:
> Why are the docstring bodies indented with 3 blank spaces instead of 4? :D

![image](https://github.com/holoviz/holoviews/assets/19758978/1b16dce5-5c60-411d-838e-9621118c28ac)